docker/common/run-key-manager.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
+# Copyright (c) The Libra Core Contributors
+# SPDX-License-Identifier: Apache-2.0
+set -e
+
+# Example Usage (arguments that are not specified will be assigned defaults):
+# ./run-key-manager.sh <IMAGE> <JSON_RPC_ENDPOINT> <ROTATION_PERIOD_SECS> <VALIDATOR_ACCOUNT> <VAULT_HOST> <VAULT_TOKEN>
+
+IMAGE="${1:-libra_common:latest}"
+JSON_RPC_ENDPOINT="${2:-http://127.0.0.1:8080}"
+ROTATION_PERIOD_SECS="${3:-3600}"
+VALIDATOR_ACCOUNT="${4:-00000000000000000000000000000000}"
```

Comment:
@davidiw, we can remove this in my next PR that will fetch the validator operator account from storage!